pages/index.tsx

```diff
@@ -139,6 +139,7 @@ const Home: FC<IHomeProps> = ({ latestPosts, network, networkSocialsData }) => {
 	// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+
```

Comment:
unncessary change, might throw linting error "not more than 1 blank line allowed"